apps/cookbook/src/app/examples/badge-example/badge-example.component.scss

```diff
@@ -1,12 +1,25 @@
 @import '~@kirbydesign/core/src/scss/utils';
+@import '~@kirbydesign/core/src/scss/utils';
+
+:host {
```

Comment:
Beautiful, consider 100% height here so the gray area extends all the way to the bottom.